src/makeRobotAccountant.js

```diff
@@ -1,12 +1,32 @@
 'use strict';
 
 /**
- *
+ * function(num1)(num2) === num1 + num2;
  * @return {function}
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let firstNumber;
+  let callNumber = 0;
+
+  const add = function(number = 0) {
+    if (firstNumber === undefined) {
+      firstNumber = number;
+
+      return add;
+    }
+
+    const sum = firstNumber + number;
+
+    firstNumber = undefined;
+    callNumber++;
+
+    return callNumber > 3 && callNumber % 2 === 0
+      ? 'Bzzz... Error!'
+      : sum;
+  };
+
+  return add;
```

Comment:
It's an overkill to create recursive function, you don't need to do it in such case.

Reply:
Your function should just add two numbers, we don't need do that recursively or infinite times

Reply:
looks like I have to use that alternative aproach ::)